modules/controller_pool/controller-primary.tpl

```diff
@@ -238,6 +238,7 @@ stringData:
     {
       "apiKey": "${equinix_api_key}",
       "projectID": "${equinix_project_id}",
+      %{ if equinix_metro > ""}"metro": "${equinix_metro}"%{ else }"facility": "${equinix_facility}"%{ endif },
```

Comment:
Looks like my change broke it:
`│ Error: failed to render : <template_file>:241,13-26: Invalid operand; Unsuitable value for left operand: a number is required., and 1 other diagnostic(s)`

Reply:
https://developer.hashicorp.com/terraform/language/expressions/operators#comparison-operators